Composer/packages/server/src/locales/en-US.json

```diff
@@ -2420,6 +2462,9 @@
   "view_kb_c382e495": {
     "message": "View KB"
   },
+  "view_on_npm_2051324d": {
+    "message": "View on npm"
+  },
   "vishwac_sena_45910bf0": {
```

Comment:
Where is this string being used? Maybe John Doe?

Reply:
It's in `src/pages/setting/extensions/ExtensionSearchResults.tsx`. It looks like we're using npm as a place for people to host extensions.